src/main/java/org/folio/circulation/domain/policy/OverdueFinePolicy.java

```diff
@@ -16,14 +56,55 @@ private OverdueFinePolicy(
   }
 
   public static OverdueFinePolicy from(JsonObject json) {
-    return new OverdueFinePolicy(
+    OverdueFinePolicy overdueFinePolicy = new OverdueFinePolicy(
       getProperty(json, "id"),
       getProperty(json, "name"),
       json.getBoolean("gracePeriodRecall"),
       json.getBoolean("countClosed")
     );
+
+    overdueFinePolicy.setOverdueFine(json.getJsonObject("overdueFine").getDouble("quantity"));
+    overdueFinePolicy.setOverdueFineInterval(OverdueFineInterval.fromValue(
+      json.getJsonObject("overdueFine").getString("intervalId")));
+    overdueFinePolicy.setMaxOverdueFine(json.getDouble("maxOverdueFine"));
+    overdueFinePolicy.setMaxOverdueRecallFine(json.getDouble("maxOverdueRecallFine"));
```

Comment:
Why are these set by mutating the object rather than as constructor parameters? Do they change during the use of the policy?

Reply:
Moved to constructor, made fields final.